package.json

```diff
@@ -12,44 +12,44 @@
     "url": "https://github.com/rdmorganiser/rdmo"
   },
   "dependencies": {
-    "@codemirror/lang-html": "^6.4.2",
-    "@codemirror/lang-javascript": "^6.1.4",
-    "@uiw/react-codemirror": "^4.19.9",
+    "@codemirror/lang-html": "^6.4.6",
+    "@codemirror/lang-javascript": "^6.2.1",
+    "@uiw/react-codemirror": "^4.21.20",
     "bootstrap-sass": "^3.4.1",
     "classnames": "^2.3.2",
     "font-awesome": "4.7.0",
-    "jquery": "^3.6.0",
-    "js-cookie": "^2.2.1",
+    "jquery": "^3.7.1",
+    "js-cookie": "^3.0.5",
     "lodash": "^4.17.21",
     "popper.js": "^1.16.1",
     "prop-types": "^15.7.2",
     "react": "^18.2.0",
-    "react-bootstrap": "~0.33.1",
```

Comment:
maybe we need to pin to,
```
"react-bootstrap": "~0.33.1",
```
since we are still on bootstrap 3, the other version are for bs4 or bs5

https://react-bootstrap-v3.netlify.app/

Reply:
Good catch!

It should be `"react-bootstrap": "0.33.1"` then?
https://www.npmjs.com/package/react-bootstrap/v/0.33.1

Reply:
yes, I think the bug is in the build, my setup was the same and I only exchanged the wheel..

Reply:
Yes, I can replicate the missing icons issue. Will look at the commits one by one to find the culprit, that hides the icons. Its not babel, that I tested this morning.
Will get back to you about this.

Reply:
could this maybe be a use-case for the e2e tests with screenshots?

Reply:
I think so.
But the tests are passing.
Playwright does not care about the missing icons.
My understanding of this feature is, that it takes a screenshot on failure.

Reply:
alright when its like that, then maybe if its possible to test for the "presence" of an icon, I could like into it later as well.